rasa/nlu/utils/hugging_face/hf_transformers.py

```diff
@@ -147,6 +185,21 @@ def _post_process_sequence_embeddings(
     def _tokenize_example(
         self, message: Message, attribute: Text
     ) -> Tuple[List[Token], List[int]]:
+        """Tokenize a single message example.
+
+        Many language models add a special char in front of (some) words and split words into
+        sub-words. To ensure the entity start and end values matches the token values,
+        tokenize the text first using the whitespace tokenizer. If individual tokens
+        are split up into multiple tokens, we make sure that the start and end value
+        of the first and last respective tokens stay the same.
+
+        Args:
+            message: Single message object to be processed.
+            attribute: Property of message to be processed, one of ``INTENT`` or ``RESPONSE``.
```

Comment:
> one of ``INTENT`` or ``RESPONSE``.

Are you sure? Not `one of ``TEXT`` and ``RESPONSE``.`?

Reply:
Ahh sorry! indeed ``TEXT`` and ``RESPONSE``

Reply:
Fixed